app/resources/en/translation/coapplicant/agreepage.json

```diff
@@ -5,8 +5,6 @@
   "subHeader": "What happens next",
   "paragraph2": "{leadExecFullName} will receive the grant of probate within 20 working days of the probate registry receiving all the necessary paperwork.",
   "paragraph3": "Contact {leadExecFullName} if you need more information.",
-  "paragraph4": "You need to sign a photocopy of the will in the top right hand corner and send it to {leadExecFullName}.",
-  "paragraph4-codicils": "You need to sign a photocopy of the will and any codicils in the top right hand corner and send it to {leadExecFullName}.",
```

Comment:
Please update the codicils/no codicils component tests for this page.

`test/component/coapplicant/testCoApplicantAgreePage.js`